### ISSUE TITLE
[Help wanted] Workflow for managing ACF fields within the theme 

### ISSUE THREAD

**Issue by e-reifer** [digitoimistodude/air-light#170]
First of all, I really like the idea behind this theme and we currently use it for our next project. So, great job in providing all these resources! 

I'm currently struggling a little bit the get the workflow up and running for creating and managing blocks with acf and would appreciate it if you could guide me in the right direction here: 

In the WP dashboard, I am unable to see the ACF page, although it is [referenced in your docs](https://github.com/digitoimistodude/air-light/wiki/Creating-your-own-Gutenberg-blocks-with-Air-light-and-Advanced-Custom-Fields). So, I thought to configure all fields with the corresponding JSON files that are created with `createblock` command.  However, I cannot find any relevant documentation for the field types and options and when creating a new empty block, I get the following (error?) message.

```
Creating locations block assets...
Importing ACF fields (json)... 
/Users/er_re/air-blocks/content/themes/air-blocks/acf-json/group_4e34afe0d4c95.json -> /var/www/nextbigproject/content/themes/mytheme/acf-json/group_4e34afe0d4c95.json
sed: 1: "s/Otsikko/Title\/
": unescaped newline inside substitute pattern
Activating ACF fields for the block... 

All done! Block created: locations
```

How would I usually go about configuring ACF fields? I would prefer to visually configure fields in the WP dashboard, which are synced to the JSON files. Is that even possible?

**Comment by ronilaukkarinen** [comment]
Thanks for using Air-light. It sounds like your issue is more about ACF and [air-block](https://github.com/digitoimistodude/air-blocks) than just Air-light theme. [air-blocks](https://github.com/digitoimistodude/air-blocks) is a tool for importing ready-made Gutenberg blocks by us via a bash script. It's still kinda under construction and not much documentation is provided about it at the moment.

As for your "unescaped newline inside substitute pattern" error in the start script, we've not seen that. Are you using macOS? Which version? The scripts are currently only supported by macOS Monterey or later.

To summarize: You can use air-light as a theme, ACF installed manually and fields configured manually.

> I would prefer to visually configure fields in the WP dashboard, which are synced to the JSON files. Is that even possible?

This is actually a default way to use ACF. You should check out ACF documentation [here](https://www.advancedcustomfields.com/resources/).

I suggest you to learn about ACF and build your Gutenberg blocks manually to a theme based on Air-light.

**Comment by e-reifer** [comment]
Okay thanks, where can I find the setting in the theme to show the ACF plugin link in the WP menu items? It seems, that for my setup this is hidden by default so users would not mess with the config. 

I've been using ACF for a few years now, so I know my way around but have not used it in this kind of environment. 

Regarding  the unescaped newline:
 
I'm running macOS Ventura 13.0 with an Intel Chip and your recommended LAMP setup.

**Comment by ronilaukkarinen** [comment]
Oh, it sounds like you are also using [air-helper](https://github.com/digitoimistodude/air-helper) plugin? We hide plugins and ACF ([see this](https://github.com/digitoimistodude/air-helper/blob/102e0047660553fd12669d014e044fb8c8e62884/plugin-helpers.php#L142)) from customer indeed so that a customer would not mess with the important stuff. In that case, look for [ACF](https://github.com/digitoimistodude/air-helper/tree/fb0980c4e5b8ac884f4fbbd129a3de82350d5189#acf) in the readme. Some of the hooks are still quite undocumented at the moment, we should put more effort to that.

As for Ventura, we have not yet tested our stack for it. Sounds like Ventura's sed command has been updated. MacOS version of sed differs a bit from the GNU one.

I hope this helps.

**Comment by e-reifer** [comment]
Yes, that's exactly what I was looking for, thanks!
